host/sgx/sgxload.c

```diff
@@ -545,24 +546,46 @@ oe_result_t oe_sgx_create_enclave(
 #elif defined(_WIN32)
 
         /* Ask OS to create the enclave */
-        DWORD enclave_error;
-        void* base = CreateEnclave(
-            GetCurrentProcess(),
-            NULL, /* Let OS choose the enclave base address */
-            secs->size,
-            secs->size,
-            ENCLAVE_TYPE_SGX,
-            (const void*)secs,
-            sizeof(ENCLAVE_CREATE_INFO_SGX),
-            &enclave_error);
+        DWORD einit_retry_count = 0;
+        while (einit_retry_count < MAX_EINIT_RETRY_COUNT)
```

Comment:
There should be a comment to indicate why we only do this retry loop on Windows and not Linux. You can lift text from #1794. You should also indicate why we think a sleep between loops is unnecessary (i.e. cost of kernel transition, as stated in the PR comments).

Reply:
ok, will incorporate.